### PR TITLE
Updated contributing.md

### PR DESCRIPTION
**Pull request by abhinavsagar**

### CODE REVIEW

**Comment by orcmid** [comment]
The update to contributing.md is useful, and it would be valuable to tie in howto's on the matter, such as what it takes to test and/or develop the code.   Is there something easy to add to the list, such as 

> acquiring and operating with the code

that could also become an useful link to how-to information.

**Comment by abhinavsagar** [comment]
@orcmid @mrkmarron Is it better now?

**Comment by orcmid** [comment]
Nice improvement, separate from what I was suggesting

**Comment by orcmid** [comment]
What's wrong with a leading topic,

`# acquiring and operating with the code`

so that folks who have never worked on a programming language or its processor can learn what the prerequisites are, and how simply to run the current build, become familiar with examples and tests?

I favor many of the expansions of this page, although it might be well to have separate pages with details and have short abstracts at this level?  

I am sorry that I give advice but do not do the work.  I am still learning how to synchronize my fork and its desktop clone, and then how to make a pull request.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
-# Contributing
+# Contributing to BosqueLanguage
+
+We love your input! We want to make contributing to this project as easy as possible, whether it's:
```
Comment (mrkmarron) [inline CONTRIBUTING.md]: Let's delete "We love you input!" here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,43 @@
...
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+
+# How Do I Submit A (Good) Bug Report?
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
```
Comment (mrkmarron) [inline CONTRIBUTING.md]: Let's include "Check if the same, or a similar, issue has already been reported" to minimize duplicates and help connect issues that may be related.

**Comment by abhinavsagar** [comment]
@mrkmarron Done the required changes.

**Pull request merged**